crates/hc_sandbox/src/run.rs

```diff
@@ -201,18 +208,8 @@ async fn check_lair_running(stdout: tokio::process::ChildStdout) {
     let _ = r.await;
 }
 
-// TODO: Find a better way to confirm the child is running.
-async fn check_started(holochain: &mut Child) {
-    let started =
-        tokio::time::timeout(std::time::Duration::from_millis(20), holochain.wait()).await;
-    if let Ok(status) = started {
-        panic!("Holochain failed to start. status: {:?}", status);
-    }
-}
-
 fn spawn_output(holochain: &mut Child, config: oneshot::Sender<u16>) {
     let stdout = holochain.stdout.take();
-    let stderr = holochain.stderr.take();
```

Comment:
Does the above mean that there cannot be an error output beside the Error arm?

Reply:
No, I've also set the stderr of the child to 'inherit' the stderr of the parent. So we'll still see stderr the same way but we don't need to have a tokio task running to read/write it. Unless we're parsing it on the way through that doesn't seem important